grakn-engine/src/main/java/ai/grakn/engine/controller/response/RelationshipType.java

```diff
@@ -51,7 +53,7 @@ public static RelationshipType create(
             @JsonProperty("plays") Set<Link> plays,
             @JsonProperty("attributes") Set<Link> attributes,
             @JsonProperty("keys") Set<Link> keys,
-            @JsonProperty("relates") Set<Link> relates){
+            @Nullable @JsonProperty("relates") Set<Link> relates){
```

Comment:
I'm more in favour of using an empty set instead of `null` (unless they have different meanings).